crud-cert-auth/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,35 +1,35 @@
 <web-app version="3.0" xmlns="http://java.sun.com/xml/ns/javaee"
-xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
-  
-  <filter>
-     <filter-name>LoggingFilter</filter-name>
-     <filter-class>com.redhat.lightblue.rest.logging.LoggingFilter</filter-class>
-  </filter>
-  <filter-mapping>
-      <filter-name>LoggingFilter</filter-name>
-      <url-pattern>/*</url-pattern>
-  </filter-mapping>
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
 
-  <security-constraint>
-    <web-resource-collection>
-      <web-resource-name>rest-crud services</web-resource-name>
-      <url-pattern>/*</url-pattern>
-    </web-resource-collection>
-    <auth-constraint>
-      <role-name>authenticated</role-name>
-    </auth-constraint>
-    <user-data-constraint>
-      <transport-guarantee>CONFIDENTIAL</transport-guarantee>
-    </user-data-constraint> 
-  </security-constraint>
+	<filter>
+		<filter-name>LoggingFilter</filter-name>
+		<filter-class>com.redhat.lightblue.rest.logging.LoggingFilter</filter-class>
+	</filter>
```

Comment:
Please use spaces, not tabs. Also, it looks like only some of the file was reformatted, which makes it hard to read.